.github/workflows/bencher-cleanup.yml

```diff
@@ -0,0 +1,20 @@
+on:
+    pull_request:
+      types: [closed]
+
+jobs:
+    archive_pr_branch:
+        name: Bencher (cloud or self-hosted) | Archive feature branch reports on PR close
+        # DO NOT REMOVE: For handling Fork PRs see Pull Requests from Forks
+        if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
```

Comment:
Just a question: is it needed to checkout the  repo for this?

Reply:
I am not sure I understand the question.
This action will run once the PR is merged (automatically ), and then it will archive the Bencher results related to this specific branch.

Reply:
My question is about the repository checkout step (the first step) of the `archive_pr_branch` job. Since that step is checking out the current branch of the repo, but the job is about archiving something in bencher cloud, I wonder if it is really needed to check out the repository first, because I have no idea if the `bencher archive` command needs the project folder to be present or not.

Reply:
I see. I am not sure it is really needed, but I have just copy-pasted this action from https://bencher.dev/docs/how-to/github-actions/, so I trust what Bencher documentation provides :)